src/agoradatatools/etl/transform/model_details.py

```diff
@@ -162,7 +163,27 @@ def transform_model_details(
     Raises:
         ValueError: If required datasets are missing or if required columns are missing from any dataset.
     """
-    check_required_datasets_and_columns(datasets, required_input)
+    if "immunohisto_measure_order" not in datasets:
+        raise ValueError(
+            "The 'immunohisto_measure_order' dataset is required but not found in datasets"
+        )
+
+    required_with_config = {
```

Comment:
Why isn't immunohisto_measure_order added to `REQUIRED_INPUT` like it is in the immunohisto transform? This extra check and adding it to the required input seems unnecessary.

Reply:
I still have the same question. Why is "immunohisto_measure_order" manually added here instead of putting it in the `REQUIRED_INPUT` declaration at the beginning of the file?